client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -19,26 +19,28 @@ import PromotionAppliesToField from './fields/promotion-applies-to-field';
 /**
  * "Coupon code" field reused for all coupon promotion types.
  */
-const couponCodeField = {
-	component: TextField,
-	labelText: translate( 'Coupon code' ),
-	explanationText: translate(
-		'Only apply this promotion when the customer supplies the coupon code'
-	),
-	placeholderText: translate( 'Enter coupon code' ),
-	isRequired: true,
-};
+const couponCodeField = (
+	<TextField
+		labelText={ translate( 'Coupon code' ) }
+		explanationText={ translate(
+			'Only apply this promotion when the customer supplies the coupon code'
+		) }
+		placeholderText={ translate( 'Enter coupon code' ) }
+		isRequired={ true }
```

Comment:
minor but could just be `isRequired`

Reply:
Replaced all `isRequired` instances, thanks